### PR TITLE
Nav/25/creating dashboard

### PR DESCRIPTION
**Pull request by ChrisWakley**
Created Nav component and styling.

https://trello.com/c/fdFtRpUi/25-create-a-nav-folder-for-dashboard

### CODE REVIEW

**Review by Zain-Ashfaq — DISMISSED** [review]
all good

**Review by Martinelli-89 — CHANGES_REQUESTED** [review]
Align text to icon.
Add transfer

File: src/components/Layout/Layout.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.layout {
```
Comment (kirsty-nology) [inline src/components/Layout/Layout.css:1]: remove all css files

File: src/pages/Nav/Nav.jsx [hunk 2]
```diff
@@ -13,7 +15,11 @@ const Nav = () => {
       </div>
 
       <div className="nav__menu">
-        <h1>Menu</h1>
+
+        <h2 className="nav__menu-header">Menu</h2>
```
Comment (kirsty-nology) [inline src/pages/Nav/Nav.jsx:19]: `className=nav__header`

File: src/pages/Nav/Nav.scss [hunk 3]
```diff
@@ -2,22 +2,28 @@
 
 .nav {
   position: absolute;
-  width: 20%;
+
```
Comment (kirsty-nology) [inline src/pages/Nav/Nav.scss:5]: remove extra lines

**Review by kirsty-nology — CHANGES_REQUESTED** [review]
All css files should be removed and consistent use of rem and variables used in sass files please

**Review by kirsty-nology — APPROVED** [review]

**Review by kirsty-nology — DISMISSED** [review]

**Review by Zain-Ashfaq — APPROVED** [review]
class

**Review by kirsty-nology — APPROVED** [review]

**Review by Martinelli-89 — APPROVED** [review]

**Pull request merged**